packages/venia-ui/lib/queries/getAvailableStoresConfigData.graphql

```diff
@@ -0,0 +1,19 @@
+query availableStoresConfigData {
```

Comment:
Instead of generic `queries` we are moving to colocating queries next to their components that utilize them.

So this would become:

```
import { GET_CONFIG_DATA_QUERY } from './storeSwitcher.gql.js';
```

And you would modify it to use `gql` and export the query.

I'd like to see this new query rewritten in this manner, but the other queries you are just modifying in this change set I leave to your discretion.

Reply:
Moved query next to storeSwitcher component.
Regarding this [issue](https://github.com/magento/pwa-studio/pull/2696#issuecomment-694329608), I've tried to merge Dave's branch into mine before, there were some conflicts, but at the first look it seems to work fine, I'm starting more thorough testing both branches merged now, and will push it, when Dave's PR will be merged.